src/scripts/main.js

```diff
@@ -1,3 +1,13 @@
 'use strict';
 
-// write your code here
+const population = document.querySelectorAll('.population');
+const totalText = document.querySelector('.total-population');
+const averageText = document.querySelector('.average-population');
+
+const numbersArray = [...population]
+  .map(number => +number.innerText.split(',').join(''));
+
+const sum = numbersArray.reduce((a, b) => a + b);
```

Comment:
not fixed commetn to provide initial value to reduce.
of course in this case it is not change result, bun it is good practise to provide in any case